test/fork/basic-functionality-tests.js

```diff
@@ -391,7 +305,8 @@ describe('basic functionality tests', function () {
         ipfsMetadata: '',
         product: {
           productType: 0,
-          yieldTokenAddress: AddressZero,
+          minPrice: 0,
+          __gap: emptyBytes(18),
```

Comment:
1. if you switch to uint144 you'll be able to just pass 0
2. does it allow passing undefined instead? as omitting it entirely?

Reply:
1. yes, that would be easier to maintain in the future.
2. no, must be defined.